src/app/modules/project/overview/overview.component.scss

```
@@ -16,264 +16,274 @@
-
-    img {
-      width: 25px;
-    }
-  }
-}
-
-.projects-table {
-  width: 100%;
-  table-layout: fixed;
-  border-collapse: separate;
-  border-spacing: 0 15px;
-
-  tr {
-    box-shadow: 0 3px 6px #0000001a;
-    cursor: pointer;
-
-    td {
-      background: $light-mode-component-color;
-    }
-
-    td:first-child {
-      border-top-left-radius: 6px;
-      border-bottom-left-radius: 6px;
-    }
-
-    td:last-child {
-      border-bottom-right-radius: 6px;
-      border-top-right-radius: 6px;
-    }
-
-    td,
-    span,
-    h3,
-    p,
-    p.large,
-    p.small {
-      margin: 0;
-      overflow: hidden;
-      text-overflow: ellipsis;
-      white-space: nowrap;
-    }
-
-    td.title {
-      width: auto;
-      display: flex;
-      align-items: center;
-      padding: 0 20px 0 0;
-
-      .icon-placeholder {
-        height: 55px;
-        width: 55px;
-        min-width: 55px;
-        margin: 10px 20px;
-        border-radius: 30px;
-        background-color: $accent-color-primary;
-        display: flex;
-        align-items: center;
-        justify-content: center;
-      }
-
-      img {
-        height: 30px;
-        max-width: 30px;
-      }
-    }
-
-    td.icon {
-      width: 20%;
-
-      svg {
-        height: 18px;
-        fill: red;
-      }
-
-      img {
-        height: 18px;
-        margin: 13px;
-      }
-    }
-  }
-}
-
-.projects-filter-wrapper{
-  min-width: 250px;
-  .projects-filter {
-    box-shadow: 0 3px 6px #0000001a;
-    background: $light-mode-component-color;
-    padding: 20px 25px;
-    border-radius: 6px;
-    margin-bottom: 20px;
-    h3 {
-      margin: 0;
-    }
-
-    .option-select {
-      margin-bottom: 15px;
-      width: 100%;
-    }
-
-    form {
-      input[type="checkbox"] {
-        display: none;
-
-        + label {
-          color: $light-mode-text-color;
-          vertical-align: middle;
-          position: relative;
-          padding-left: 20px;
-
-          &::before,
-          &::after {
-            content: "";
-            display: inline-block;
-            border-radius: 2px;
-            position: absolute;
-          }
-
-          &::before {
-            width: 15px;
-            height: 15px;
-            border: 1px $light-mode-border-color solid;
-            background: $light-mode-component-color;
-            left: 0;
-            top: calc(50% - 15px / 2);
-          }
-        }
-
-        &:checked {
-          + label {
-            &::after {
-              width: 9px;
-              height: 9px;
-              background: $accent-color-primary;
-              position: absolute;
-              top: calc(50% - 9px / 2);
-              left: 3px;
+ @import "assets/styles/variables";
+
+ .pagination-div {
+   margin-left: 35%;
+ }
+ 
+ .pagination-footer::ng-deep .page-link {
+   z-index: 1;
+   color: #323232;
+ }
+ 
+ .pagination-footer::ng-deep .page-item.active .page-link {
+   border-color: #e85b3f;
+   background-color: #e85b3f !important;
```

```suggestion
   background-color: #e85b3f;
```
Since ng-deep is being used, !important is not needed.